core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -127,6 +131,9 @@ object LogConfig {
   val LeaderReplicationThrottledReplicasProp = "leader.replication.throttled.replicas"
   val FollowerReplicationThrottledReplicasProp = "follower.replication.throttled.replicas"
 
+  /** ********* Id mapping snapshots ****************/
+  val MaxIdMapSnapshotsProp = "max.id.map.snapshots"
```

Comment:
Hmm. It is not part of the KIP. We agreed that 2 would be a good static value, I think. Will update.